src/physics/em/BetheHeitlerModel.cc

```diff
@@ -0,0 +1,76 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BetheHeitlerModel.cc
+//---------------------------------------------------------------------------//
+#include "BetheHeitlerModel.hh"
+
+#include "base/Assert.hh"
+#include "physics/base/PDGNumber.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct from model ID and other necessary data.
+ */
+BetheHeitlerModel::BetheHeitlerModel(ModelId               id,
+                                     const ParticleParams& particles)
+{
+    CELER_EXPECT(id);
+    interface_.model_id    = id;
+    interface_.electron_id = particles.find(pdg::electron());
+    interface_.positron_id = particles.find(pdg::positron());
+    interface_.gamma_id    = particles.find(pdg::gamma());
+
+    CELER_VALIDATE(interface_.electron_id && interface_.positron_id
+                       && interface_.gamma_id,
+                   "Electron, positron and gamma particles must be enabled to "
+                   "use the "
+                   "Bethe-Heitler Model.");
```

Comment:
This sort of thing is an artifact of clang-format being able to split string constants but not recombine them 🙄 
```suggestion
                   "use the Bethe-Heitler Model.");
```